autoload/battery/backend/freebsd.vim

```diff
@@ -0,0 +1,53 @@
+" Reference:
+" sysctl -d hw.acpi.battery
+"
+" hw.acpi.battery: battery status and info
+" hw.acpi.battery.info_expire: time in seconds until info is refreshed
+" hw.acpi.battery.units: number of batteries
+" hw.acpi.battery.state: current status flags
+"   discharging = 1
+"   charging = 2
+"   critical = 4
+" hw.acpi.battery.rate: present rate in mW
+" hw.acpi.battery.time: remaining time in minutes
+" hw.acpi.battery.life: percent capacity remaining
+
+let s:bat_status = 'hw.acpi.battery.state'
+let s:bat_capacity = 'hw.acpi.battery.life'
+
+let s:Job = vital#battery#import('System.Job')
+
+let s:job = v:null
```

Comment:
It's minor but

1. Please use UPPER_CAMEL_CASE for constant variables
2. Reorder definitions to Vital modules, constant variables, script local variables

like

```suggestion
let s:Job = vital#battery#import('System.Job')

let s:BAT_STATUS = 'hw.acpi.battery.state'
let s:BAT_CAPACITY = 'hw.acpi.battery.life'

let s:job = v:null
```

Reply:
done